asset/css/variables.less

```diff
@@ -162,5 +182,13 @@
     --suggestions-relation-path-focus-bg: var(--base-gray);
 
     --card-border-color: var(--base-gray-light);
+
+    --schedule-element-fields-bg: var(--default-input-bg);
+    --schedule-element-default-text-color: var(--base-primary-color);
+
+    --schedule-element-fields-hover-bg: var(--base-primary-dark);
+
+    --schedule-element-fields-border-color: var(--base-gray-light);
+    --schedule-element-fields-disabled-bg: var(--base-gray-lighter);
```

Comment:
I still miss the following definitions:

* schedule-element-text-color
* schedule-element-fields-selected-bg
* schedule-element-fields-selected-hover-bg
* schedule-element-fields-disabled-color
* schedule-element-fields-disabled-selected-bg

Remember, if a LESS variable is defined with a base/default variable in dark mode, the same must be done in light mode.

Reply:
✅